### PR TITLE
Analytics additions

### PR DESCRIPTION
**Pull request by ebrodymoore**
Added hubspot data quality to 'sales' and created bizops/analytics documentation

### CODE REVIEW

File: handbook/bizops/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- When do you need the deliverable by? 
+- Is this request a nice-to-have or a necessity?
+
+**Small asks and questions:** Post in the #analytics channel in Slack. 
+
+## Data sources
+
+Here are the following sources we collect data from:
+
+* [Looker](#using-looker): Business intelligence/data visualization tool
```
Comment (dadlerj) [inline handbook/bizops/index.md]: Does this belong in a "data _sources_" list? Perhaps there's a sources list, and a tools list?

File: handbook/bizops/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* Sourcegraph.com Site-admin pages: customer subscriptions and license keys
+* [Pings](https://docs.sourcegraph.com/admin/pings) from self-hosted Sourcegraph instances containing anonymous and aggregated information
+* [Custom tool to track events](https://github.com/sourcegraph/sourcegraph/issues/5486) on the Sourcegraph.com instance
+
+### Using Looker
+
+## Sourcegraph quick links
+
+[All Instances](https://sourcegraph.looker.com/looks/436)<br/>
+[Specific Instance Overview](https://sourcegraph.looker.com/dashboards/94?Unique%20Server%20ID=Uber&Site%20ID=&filter_config=%7B%22Unique%20Server%20ID%22:%5B%7B%22type%22:%22%3D%22,%22values%22:%5B%7B%22constant%22:%22Uber%22%7D,%7B%7D%5D,%22id%22:4%7D%5D,%22Site%20ID%22:%5B%7B%22type%22:%22%3D%22,%22values%22:%5B%7B%22constant%22:%22%22%7D,%7B%7D%5D,%22id%22:5%7D%5D%7D)<br/>
```
Comment (dadlerj) [inline handbook/bizops/index.md]:
```suggestion
[Specific Instance Overview](https://sourcegraph.looker.com/dashboards/94?Unique%20Server%20ID=Uber&Site%20ID=&filter_config=%7B%22Unique%20Server%20ID%22:%5B%7B%22type%22:%22%3D%22,%22values%22:%5B%7B%22constant%22:%%7D,%7B%7D%5D,%22id%22:4%7D%5D,%22Site%20ID%22:%5B%7B%22type%22:%22%3D%22,%22values%22:%5B%7B%22constant%22:%22%22%7D,%7B%7D%5D,%22id%22:5%7D%5D%7D)<br/>
```

Please test a url that doesn't come with any pre-filled options...

File: handbook/bizops/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Folders:<br/>
+[Marketing](https://sourcegraph.looker.com/folders/109)<br/>
+[Sales](https://sourcegraph.looker.com/folders/114)<br/>
+[Product Insights](https://sourcegraph.looker.com/folders/113)<br/>
+
+## Getting Started with Looker
+
+Looker enables us to explore and visualize [Sourcegraph data sources](https://docs.google.com/document/d/1H_OFThjtOAnMVxH6vxUnS_NUMk0ktH1KuM0aJ-8kciU/edit?usp=sharing). If you're a new user, here's where to get started.
```
Comment (dadlerj) [inline handbook/bizops/index.md]: Update URL

File: handbook/sales/index.md [hunk 4]
```diff
@@ -114,3 +114,43 @@ Different customers and deals have different needs from our sales team. We segme
...
 - **Tier 2** = $42k+ ARR
 - **Tier 3** = $22k+ ARR
+
+## Using HubSpot
+
+This page describes how we use HubSpot to maintain high data quality. This allows us to increase the effectiveness of all Sourcegraph teams through accurate insights. 
+
+Account Executives are responisble for maintaining HubSpot as a [source of truth](https://about.sourcegraph.com/handbook/communication#sources-of-truth). The data is synced between HubSpot and Looker every weekend so this data must be updated (at least) by end of day Friday. 
+
+### Associating Contacts to Deals
```
Comment (dadlerj) [inline handbook/sales/index.md]: In general, you should use sentence case (versus pascal case) per the style guide: https://about.sourcegraph.com/handbook/communication/style_guide

File: handbook/sales/index.md [hunk 5]
```diff
@@ -114,3 +114,43 @@ Different customers and deals have different needs from our sales team. We segme
...
+
+It is critical to associate all of the most important contacts within a company with any new deal. This should include the technical decision-maker, the economic decision-maker (if they are different) and the original member who introduced Sourcegraph to the organization. 
+
+This allows us to evaluate the effectiveness of marketing channels and sales touchpoints that our team has with an organization. How we reached the person(s) who introduced Sourcegraph to their organization is one of the most important factors in evaluating the success of marketing activities.
```
Comment (dadlerj) [inline handbook/sales/index.md]: It also ensures that all deal-related communication is visible in the deal timeline view! Perhaps worth adding, as it's a benefit that accrues directly to the AEs who have to do this work.

File: handbook/sales/index.md [hunk 6]
```diff
@@ -114,3 +114,43 @@ Different customers and deals have different needs from our sales team. We segme
...
+
+### Maintaining up-to-date information on deals
```
Comment (dadlerj) [inline handbook/sales/index.md]: Should this section say something like "Please keep the following information updated for deals:"? Just jumping into a list is a bit confusing.

**Comment by dadlerj** [comment]
Ah, and it looks like this is failing CI:

image

Try to click through to see where the issue is. You may need a Netlify user account (request from Ryan or Farhan I believe...)

**Comment by sqs** [comment]
@ebrodymoore This PR introduced some broken links that CI caught (see https://github.com/sourcegraph/about/pull/365#issuecomment-558018925). You have 2 links to:

- `[BizOps](@sourcegraph/bizops)`
- `[BizOps](@sourcegrapgh/bizops)` (typo)

These `@sourcegraph/bizops` are not valid URLs. GitHub will link `@sourcegraph/bizops` if you type that text by itself, but GitHub and our handbook site doesn't understand them as links.

I will fix this and document how to avoid this problem in the future. Will tag you in the PR.

File: handbook/bizops/index.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+### Analytics
```
Comment (sqs) [inline handbook/bizops/index.md]: @ebrodymoore The top-level header (largest) is `#` and the 2nd-level header (smaller) is `##`. I think you got these reversed.

**Pull request merged**